mono_repo/test/check_comand_test.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:test/test.dart';
+
+import 'package:mono_repo/src/check_command.dart';
+import 'package:mono_repo/src/pubspec.dart';
+import 'package:test_descriptor/test_descriptor.dart' as d;
+
+main() {
+  test('check', () async {
+    await d.dir('foo', [
+      d.file('pubspec.yaml', r'''
+name: foo
+
+dependencies:
+  build: any
+''')
+    ]).create();
+
+    await d.dir('bar', [
+      d.file('pubspec.yaml', r'''
+name: bar
+
+dependencies:
+  build:
+    git:
+      url: https://github.com/dart-lang/build.git
```

Comment:
Would be good to have a test with one of these fields missing as well (pretty sure it would pass anyways)

Reply:
Have a test where we check with/without build and ref.

I'm just trying to support what pub supports. Pub will blow up if there is no URL, for instance